streetcrm/static/css/streetcrm.css

```diff
@@ -533,14 +533,14 @@ ul#id_group {
     padding-left: 0;
 }
 
-ul#id_time_1,
+ul#id_time_2,
```

Comment:
I didn't understand this part of the change.  Why are we changing `id_time_1` to `id_time_2` here and below, but then adding a selector for `id_time_1` below?

Reply:
So, that has to do with how django admin was ordering field names based on how their widgets all magically worked.  I basically went in to the inspector and looked at how it was naming things and added css accordingly.  For some reason, it reordered stuff.  Django is magical.

Reply:
I'm just looking at this from the point of view of the diff:

I see `id_time_1` get replaced with `id_time_2` in `streetcrm/static/css/streetcrm.css`.  This is a replacement: afterwards, `id_time_1` no longer exists (at least in that file).

Then elsewhere in the diff I see

```
select#id_time_1 {
    float: left;
}
```

get added, even though `id_time_1` no longer exists thanks to the earlier hunk.  Meanwhile, nowhere else in the diff is there a corresponding change in an HTML or template file in which an ID's name changes.

Django may or may not be magical, but just from a code level, I'm not understanding how this part of the diff can even be correct.  I grant that it *could* be, but some explanation is required.  Just from looking at the diff, it appears wonky.

Reply:
Right, because the TimeTemplateField generates html dynamically based on Django being magical.  That html got reordered, so that the id_time_1 was, instead of being the radio select (a ul with radios inside of it), now a dropdown representing the minutes.  id_time_2 was now the radio select.  So I have to change the css to move `id_time_1 li` to `id_time_2 li` and make a new entry for `id_time_1` for the new minutes select.

Assumedly, if the minutes select had been placed to the left of rthe hours select, it would have been `id_time_0` and if it had been placed to the right of the am/pm switch, it would have been `id_time_2`